tests/tests/swfs/avm2/sound_rootless/Test.as

```diff
@@ -0,0 +1,42 @@
+﻿package {
+    import flash.display.MovieClip;
+    import flash.display.Stage;
+    import flash.media.Sound;
+    import flash.events.Event;
+    import flash.net.URLRequest;
+
+    public class Test extends MovieClip {
+        var actualStage:Stage;
+
+
+        public function Test() {
+    		    actualStage = stage;
+
+            trace("Initial state");
+
+    			  stage.removeChildAt(0);
+
+    			  trace("Removed root");
+
+            var sound_url:URLRequest = new URLRequest("file://noise.mp3");
```

Comment:
This is incorrect, it should be `new URLRequest("noise.mp3");`

Reply:
changed `"file://noise.mp3"` to `"noise.mp3"`